content/docs/styled-system/color-mode.mdx

```diff
@@ -80,36 +90,75 @@ export default theme
 > Remember to pass your custom `theme` to the `ChakraProvider`, otherwise your
 > color mode config won't be taken into consideration.
 
-#### Behavior of ColorMode
+#### Common Configurations
 
-The current hierarchy of how the color mode is defined is as follows:
+To help you define you desired behavior, here we define all common usage of the theme config :
```

Comment:
Remove the empty space between the last g and the `:`
```suggestion
To help you define you desired behavior, here we define all common usage of the theme config:
```

Reply:
Included in commit 1d1ec8bb6a4a658357504dbde6b8077c8ac6cfb0